components/automate-ui/src/app/helpers/auth/authorized.ts

```diff
@@ -60,14 +62,28 @@ export class AuthorizedChecker {
       this.store.dispatch(new GetUserParamPerms(payload)));
   }
 
+  private inflate(inputs: CheckObj[]): CheckObj[] {
```

Comment:
this is maybe the worst nit i've ever dared to nit, but the word `inflate` keeps feeling obtuse. I don't know if `expanded` is better or if there is an option to reveal a little of what it's being expanded _to_ ...

Reply:
`parameterizedEndpoint`?

Reply:
Interesting choice there, @blakestier. In original introspection (back-end) code (a year or more ago) I used "expanded". Here I tried "inflated" to see if it had better imagery. Could change to "expand" if you prefer.